src/routes/activity/repository.js

```diff
@@ -0,0 +1,25 @@
+const { db, errors } = require('../../shared')
+
+const { InsertActivityError } = errors.system
+
+const insertActivity = async (activity) => {
+    const { idAlternative, user_id, device_id, device_serial, idQuestion,  } = activity
```

Comment:
Here you can divide the destruction operator.
```suggestion
    const { idAlternative, dQuestio,  } = activity
    const { userId, deviceSerial, deviceId } = auth
```